client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
```

Comment:
The `.form-button` class and `.reset-password-confirm-form__button` should be considered of equal precedence to the CSS engine, so if using only the `.reset-password-confirm-form__button` selector it would be up to the order in which the CSS imports were included